controller/admin_message/manager.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, List
+from datetime import datetime
+from submodules.model.business_objects import admin_message
+from submodules.model.models import AdminMessage
+from util.notification import send_global_update_for_all_organizations
+
+
+def get_all_admin_messages() -> List[AdminMessage]:
+    get_all_active_admin_messages()  # set messages on archived if they should be archived
+    return admin_message.get_all()
+
+
+def get_all_active_admin_messages() -> List[AdminMessage]:
+    messages = admin_message.get_all_active()
+
+    messages_to_return = []
+    now = datetime.now()
+
+    for message in messages:
+        if now > message.archive_date:
+            admin_message.archive(
+                message.id, None, now, "Archive date reached.", with_commit=True
+            )
+            send_global_update_for_all_organizations(f"admin_message")
```

Comment:
this sends an update for every message that is archived instead of sending one if any is archived.

also f string without interpolation

Reply:
Done